docker/mongodb-kubernetes-tests/tests/search/search_community_external_mongod_basic.py

```diff
@@ -0,0 +1,144 @@
+
+USER_NAME = "mdb-user"
+USER_PASSWORD = "mdb-user-pass"
+
+MDBC_RESOURCE_NAME = "mdbc-rs"
+MDBS_RESOURCE_NAME = "mdbs"
+
+
+@fixture(scope="function")
+def mdbc(namespace: str) -> MongoDBCommunity:
+    resource = MongoDBCommunity.from_yaml(
+        yaml_fixture("community-replicaset-sample-mflix-external.yaml"),
+        name=MDBC_RESOURCE_NAME,
+        namespace=namespace,
+    )
+
+    if try_load(resource):
+        return resource
+
+    mongot_host = f"{MDBS_RESOURCE_NAME}-search-svc.{namespace}.svc.cluster.local:27027"
+    if "additionalMongodConfig" not in resource["spec"]:
+        resource["spec"]["additionalMongodConfig"] = {}
+    if "setParameter" not in resource["spec"]["additionalMongodConfig"]:
+        resource["spec"]["additionalMongodConfig"]["setParameter"] = {}
+
+    resource["spec"]["additionalMongodConfig"]["setParameter"].update(
+        {
+            "mongotHost": mongot_host,
+            "searchIndexManagementHostAndPort": mongot_host,
+            "skipAuthenticationToSearchIndexManagementServer": False,
+            "searchTLSMode": "disabled",
+        }
+    )
+
+    return resource
+
+
+@fixture(scope="function")
+def mdbs(namespace: str, mdbc: MongoDBCommunity) -> MongoDBSearch:
+    resource = MongoDBSearch.from_yaml(
+        yaml_fixture("search-minimal.yaml"),
+        name=MDBS_RESOURCE_NAME,
+        namespace=namespace,
+    )
+
+    seeds = [
+        f"{mdbc.name}-{i}.{mdbc.name}-svc.{namespace}.svc.cluster.local:27017" for i in range(mdbc["spec"]["members"])
+    ]
+
+    resource["spec"] = {
+        "source": {
+            "external": {
+                "hostAndPorts": seeds,
+                "keyFileSecretRef": {"name": f"{mdbc.name}-keyfile", "key": "keyfile"},
+                "tls": {"enabled": False},
+            },
+            "passwordSecretRef": {"name": f"{MDBC_RESOURCE_NAME}-{MONGOT_USER_NAME}-password", "key": "password"},
+            "username": MONGOT_USER_NAME,
+        }
+    }
+
+    return resource
+
+
+@mark.e2e_search_external_basic
+def test_install_operator(namespace: str, operator_installation_config: dict[str, str]):
+    operator = get_default_operator(namespace, operator_installation_config=operator_installation_config)
+    operator.assert_is_running()
+
+
+@mark.e2e_search_external_basic
+def test_install_secrets(namespace: str, mdbs: MongoDBSearch):
+    create_or_update_secret(namespace=namespace, name=f"{USER_NAME}-password", data={"password": USER_PASSWORD})
+    create_or_update_secret(
+        namespace=namespace, name=f"{ADMIN_USER_NAME}-password", data={"password": ADMIN_USER_PASSWORD}
+    )
+
+    create_or_update_secret(
+        namespace=namespace,
+        name=f"{MDBC_RESOURCE_NAME}-{MONGOT_USER_NAME}-password",
+        data={"password": MONGOT_USER_PASSWORD},
+    )
+
+
+@mark.e2e_search_external_basic
+def test_create_database_resource(mdbc: MongoDBCommunity):
+    mdbc.update()
+    mdbc.assert_reaches_phase(Phase.Running, timeout=1000)
```

Comment:
@fealebenpae Following up on what we discussed during the meeting:
I ran a honeycomb query to check the time needed for reaching desired phases, you can find it here: https://ui.honeycomb.io/mongodb-4b/environments/production/board-query/zTB3haUQqBZ/result/kWYY4MaAcoP
It seems that tests never need more than ~250s to reach running phase, even less depending on the exact function.

I'll double check that in the logs (because our tests automatically log how long it took to reach an asserted phase), to make sure I ran the correct query.
But I think we can reduce them

Reply:
Please let me know when you address this comment 🙏 This is the only "blocking" one for me, although I'll leave my LGTM